doc/index.rst

```diff
@@ -56,7 +56,6 @@ Note: if you are using a Pachyderm version < 1.4, you can find relevant docs `he
     deployment/custom_object_stores
     deployment/aws_cloudfront
     deployment/upgrading
-    deployment/migrations
```

Comment:
Just removing from the index without removing the actual file? that’s probably fine for now although that does mean it’s googleable long term and we’ll want to fix that. Perhaps add a note at the top that says “migration docs have been migrated to a new location: LINK”

Reply:
Spent some time how to put an edit of another file in the same PR.  I removed the old content, since it's in the new file anyway, and left it with one line, below:

This content has been moved to the new section, [Backup, Restore, and Migrate](../managing_pachyderm/backup_restore_and_migrate.html)